ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/CopyAddressHeader.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveCore
+import DesignSystem
+import Strings
+import SwiftUI
+
+/// List header for displaying text with `...` menu to copy address.
+/// Bitcoin accounts will only display `...` menu when `btcAccountInfo`
+/// is populated, allowing copy of `nextReceiveAddress`.
+struct CopyAddressHeader: View {
+
+  let displayText: String
+  let account: BraveWallet.AccountInfo
+  let btcAccountInfo: BraveWallet.BitcoinAccountInfo?
+
+  init(
+    displayText: String,
+    account: BraveWallet.AccountInfo,
+    btcAccountInfo: BraveWallet.BitcoinAccountInfo?
+  ) {
+    self.displayText = displayText
+    self.account = account
+    self.btcAccountInfo = btcAccountInfo
+  }
+
+  var body: some View {
+    HStack {
+      Text(displayText)
+      Spacer()
+      if account.coin == .btc {
+        if let btcAccountInfo {
+          addressMenu(for: btcAccountInfo.nextReceiveAddress.addressString)
```

Comment:
Why that should "nextReceiveAddress". 
Desktop UI shows next address for "Receive BTCs screen" when user explicilty requests and address for funds.
I see in screenshots that this new address appears for unrelated transaction confirmation screen which might be confusing for users.
cc: @jamesmudgett

Reply:
> I see in screenshots that this new address appears for unrelated transaction confirmation screen which might be confusing for users.

The address that appears in Transaction Confirmation is the `To` address in the BTC send transaction (from BtcTxData `to` property), the same as desktop displays the to address. We are not fetching the next receive address for the from account, only the account name is displayed for the from account.

Reply:
@supermassive can you take another look here? Next receive address is not used on transaction confirmation screen (it's only used in `Select a Token to Send` modal until iOS has `To address` drop down support similar to desktop):

https://github.com/brave/brave-core/assets/5314553/a49ed482-ced8-4937-83d3-a1571495975d

Desktop for comparison:
<img width="394" alt="Screenshot 2024-05-17 at 3 06 02 PM" src="https://github.com/brave/brave-core/assets/5314553/708ca810-c556-4e3a-ae22-e98399bcbb23"> | <img width="649" alt="Screenshot 2024-05-17 at 3 06 09 PM" src="https://github.com/brave/brave-core/assets/5314553/f136caf2-28d1-4e03-985f-ad729622f2ce">
--|--

Reply:
video lgtm
thanks!